include/revng/Model/Argument.h

```diff
@@ -42,8 +42,7 @@ class model::Argument : public model::generated::Argument {
   Identifier name() const;
 
 public:
-  bool verify() const debug_function;
-  bool verify(bool Assert) const debug_function;
+  bool verify(bool Assert = false) const debug_function;
```

Comment:
Does `p Lol->verify()` still works in `gdb`? If not, I'm against this change.

`debug_function`s are designed to be called from `gdb`/`lldb` and should have the dumbest possible interface.

Reply:
Ok, good catch, `p Lol->verify(false)` works but `p Lol->verify()` doesn't. I guess this boilerplate wasn't as useless as I thought.